### DISCUSSION TITLE
How can I fetch data about all of my historical sensor ticks?

### DISCUSSION THREAD

**Discussion by dagsir** [dagster-io/dagster#17995] [Q&A] [answered]
Hi Dagster Team, is there a table in the metadata db that contains data for all sensor ticks? `job_ticks` and `event_logs` seem to only contain runs that successfully trigger the job specified in the sensor.

---

*The question was originally asked in [Dagster Slack](https://dagster.slack.com/archives/C01U954MEER/p1699943380378639?thread_ts=1699943380.378639&cid=C01U954MEER).*

**Comment by alangenfeld** [comment, marked as answer]
The `job_ticks` table stores information for each tick of a sensor/schedule. This information is set-up to be removed over time using these settings https://docs.dagster.io/deployment/dagster-instance#data-retention

So what you see left in the database will reflect your retention settings.

**Reply by fdt199** [reply]
when does dagster trigger the deletion of data in job_ticks, I don't see documentation describing this
